cg/services/delivery_message/messages/utils.py

```diff
@@ -10,13 +11,12 @@ def get_caesar_delivery_path(case: Case) -> str:
 def get_scout_link(case: Case) -> str:
     customer_id: str = case.customer.internal_id
     case_name: str = case.name
-    return f"https://scout.scilifelab.se/{customer_id}/{case_name}"
-
-
-def get_scout38_link(case: Case) -> str:
-    customer_id: str = case.customer.internal_id
-    case_name: str = case.name
-    return f"https://scout38.scilifelab.se/{customer_id}/{case_name}"
+    url: str = (
+        "https://scout38.sys.scilifelab.se/"
```

Comment:
💯 

We will obviously need some more logic here later, with more pipes on 38, but you know this already - one thing at the time!

Reply:
exactly 😄

Reply:
Compare https://github.com/Clinical-Genomics/IT-issues/issues/901